src/ActionsImporter/Commands/AzureDevOps/BuildPipeline/DryRun.cs

```diff
@@ -11,5 +11,15 @@ public DryRun(string[] args) : base(args)
 
     protected override string Name => "pipeline";
     protected override string Description => "Target a designer or YAML pipeline";
-    protected override ImmutableArray<Option> Options => ImmutableArray<Option>.Empty;
+
+    public static readonly Option<int> PipelineId = new(new[] { "--pipeline-id", "-i" })
```

Comment:
should the changes in this file be applied to `src/ActionsImporter/Commands/AzureDevOps/DryRun.cs` instead and be reverted here?

Reply:
no because that is for both build _and_ release no?

Reply:
oh yeah I thought that was a valid option for both.